src/devices/acurite.c

```diff
@@ -445,7 +447,20 @@ static int acurite_6045_decode (bitrow_t bb, int browlen) {
 	printf("\n");
     }
 
+    data = data_make(
+       "time",			"",			DATA_STRING,	time_str,
+       "model",	        	"",			DATA_STRING,	"Acurite Lightning 6045M",
+       "sensor_id",    		NULL,  			DATA_FORMAT,    "0x%02X",   DATA_INT,       sensor_id,
+       "channel",  		NULL,     		DATA_STRING, 	&channel_str,
```

Comment:
FYI, that's a bug.  Looks like you might have tried, &channel, but that won't work because there is DATA_CHAR, so you need a null terminated string.  so you changed back to channel_str, but left the address of operator.   I just happened to notice I was getting the wrong channel.